packages/portal/src/components/Contents/NewDatasetsPageContent.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Link from 'next/link';
+import { Text } from '@mantine/core';
+import CardGrid, { CardGridItem } from '../CardGrid';
+import MultiPartText, { MultiPartTextPart } from '../MultiPartText';
+
+
+export interface NewDatasetsPageConfig {
+    readonly title: string;
+    readonly introduction: ReadonlyArray<MultiPartTextPart>;
+    readonly sections: ReadonlyArray<{
+        title: string; introduction: string; cards: ReadonlyArray<CardGridItem>;
+    }>;
+}
+
+
+
+const NewDatasetsPageContent = ({title, introduction, sections}: NewDatasetsPageConfig) => {
+  return (
+    <React.Fragment>
+      <div className='py-10 text-sm px-10'>
+        <Link legacyBehavior href={'/'}>Home&nbsp;</Link>
+        {' > '}
+        <span className='text-heal-primary'>&nbsp;{title}</span>
+      </div>
+      <div className='text-md pl-10'>
+        <Text className='font-bold text-4xl text-gen3-coal font-montserrat pb-8'>{title}</Text>
+        <div className='pl-5'>
+          <MultiPartText parts={introduction}/>
+          {
+            (sections || []).map(
+              ({title, introduction, cards}, i) => {
+                return (
+                  <div key={i} className ='mt-10'>
+                    {/* <Text className='font-bold text-3xl text-gen3-coal font-montserrat pb-8'>{title}</Text> */}
```

Comment:
Let's remove this if not needed.
Or better, we should change `ResourcePageContent.tsx` so that this component will only be rendered if `sections.title` exists. That way we don't need to duplicate this code for adding an extremely similar page.
If we do that, teh we should also rename `ResourcePageContent` to something more generic